### ISSUE TITLE
More detailed version of Tocker Trials

### ISSUE THREAD

**Issue by xaiyadev** [developers192/DetailedLoLRPC#21]
I would love if there would be more information like on what stage you are on or what score you have.

**Comment by developers192** [comment]
I personally haven't tried the mode yet, but given that it's a time-limited mode, I don't think it's worth implementing. Maybe in the future I might add more details like ranks and kda to all modes, but probably not anytime near though

**Comment by daglaroglou** [comment]
you can do KDA, simply throw a get request on https://127.0.0.1:2999/liveclientdata/playerscores?summonerName=XXX#0000

**Comment by daglaroglou** [comment]
id love to be part of it if you want as well

**Comment by developers192** [comment]
If you want sure, you can do pull requests or sth, I don't really know how github works. Besides, my code is really messy as this was my first ever project so if you can work it out i'd be very surprised xd

**Comment by daglaroglou** [comment]
yeah i dont know how prs works either... do you got discord or something by any chance so i can send codes and stuff?

**Comment by developers192** [comment]
sure add me on discord @11shadows
